scripts/portstat

```diff
@@ -16,304 +16,200 @@ import re
 import subprocess
 import sys
 import time
+import redis
 
 from collections import namedtuple, OrderedDict
+from sonic_py import sonic_py
 from tabulate import tabulate
 
-BCMCMD_PATH = "/usr/bin/bcmcmd"
-NETSTAT_PATH = "/bin/netstat"
-CAT_PATH = "/bin/cat"
-
 PORT_RATE = 40
 
 NStats = namedtuple("NStats", "rx_ok, rx_err, rx_drop, rx_ovr, tx_ok,\
                     tx_err, tx_drop, tx_ovr, rx_byt, tx_byt")
 header = ['Iface', 'RX_OK', 'RX_RATE', 'RX_UTIL', 'RX_ERR', 'RX_DRP', 'RX_OVR',
           'TX_OK', 'TX_RATE', 'TX_UTIL', 'TX_ERR', 'TX_DRP', 'TX_OVR']
 
-port_intf_dict = {
-    "xe0": "Ethernet0",
-    "xe1": "Ethernet4",
-    "xe2": "Ethernet8",
-    "xe3": "Ethernet12",
-    "xe4": "Ethernet16",
-    "xe5": "Ethernet20",
-    "xe6": "Ethernet24",
-    "xe7": "Ethernet28",
-    "xe8": "Ethernet32",
-    "xe9": "Ethernet36",
-    "xe10": "Ethernet40",
-    "xe11": "Ethernet44",
-    "xe12": "Ethernet48",
-    "xe13": "Ethernet52",
-    "xe14": "Ethernet56",
-    "xe15": "Ethernet60",
-    "xe16": "Ethernet64",
-    "xe17": "Ethernet68",
-    "xe18": "Ethernet72",
-    "xe19": "Ethernet76",
-    "xe20": "Ethernet80",
-    "xe21": "Ethernet84",
-    "xe22": "Ethernet88",
-    "xe23": "Ethernet92",
-    "xe24": "Ethernet96",
-    "xe25": "Ethernet100",
-    "xe26": "Ethernet104",
-    "xe27": "Ethernet108",
-    "xe28": "Ethernet112",
-    "xe29": "Ethernet116",
-    "xe30": "Ethernet120",
-    "xe31": "Ethernet124"
-}
-
-filter_bucket_dict = {
-    'RPKT': 0,
-    'RFCR': 1,
-    'RJBR': 1,
-    'RDBGC0': 2,
-    'DROP_PKT_ING': 2,
-    'ROVR': 3,
-    'TPKT': 4,
-    'TERR': 5,
-    'TFCS': 5,
-    'TJBR': 5,
-    'TRPKT': 5,
-    'TDBGC0': 6,
-    'TDBGC3': 6,
-    'TDBGC5': 6,
-    'TDBGC8': 6,
-    'UCQ_DROP_PKT(0)': 6,
-    'UCQ_DROP_PKT(1)': 6,
-    'UCQ_DROP_PKT(2)': 6,
-    'UCQ_DROP_PKT(3)': 6,
-    'UCQ_DROP_PKT(4)': 6,
-    'UCQ_DROP_PKT(5)': 6,
-    'UCQ_DROP_PKT(6)': 6,
-    'UCQ_DROP_PKT(7)': 6,
-    'UCQ_DROP_PKT(8)': 6,
-    'UCQ_DROP_PKT(9)': 6,
-    'UCQ_DROP_PKT(10)': 6,
-    'UCQ_DROP_PKT(11)': 6,
-    'TOVR': 7,
-    'RBYT': 8,
-    'TBYT': 9
+counter_bucket_dict = {
+    'SAI_PORT_STAT_IF_IN_UCAST_PKTS': 0,
+    'SAI_PORT_STAT_IF_IN_NON_UCAST_PKTS': 0,
+    'SAI_PORT_STAT_IF_IN_ERRORS': 1,
+    'SAI_PORT_STAT_IF_IN_DISCARDS': 2,
+    'SAI_PORT_STAT_ETHER_TX_OVERSIZE_PKTS': 3,
+    'SAI_PORT_STAT_IF_OUT_UCAST_PKTS': 4,
+    'SAI_PORT_STAT_IF_OUT_NON_UCAST_PKTS': 4,
+    'SAI_PORT_STAT_IF_OUT_ERRORS': 5,
+    'SAI_PORT_STAT_IF_OUT_DISCARDS': 6,
+    'SAI_PORT_STAT_ETHER_RX_OVERSIZE_PKTS': 7,
+    'SAI_PORT_STAT_IF_IN_OCTETS': 8,
+    'SAI_PORT_STAT_IF_OUT_OCTETS': 9
 }
 
+COUNTER_TABLE_PREFIX = "COUNTERS:"
+
+def natural_sort(l):
+    convert = lambda text: int(text) if text.isdigit() else text.lower()
+    alphanum_key = lambda key: [ convert(c) for c in re.split('([0-9]+)', key) ]
+    return sorted(l, key = alphanum_key)
```

Comment:
I don't like natsort library because:
1. debian has outdated version of the library
2. this library break an api
Also I think this function is so small so it's ok to have it in our code.